packages/experimental/Divider/src/Divider.types.ts

```diff
@@ -0,0 +1,93 @@
+import type { ViewProps, ColorValue } from 'react-native';
+import type { IconPropsV1 as IconProps } from '@fluentui-react-native/icon';
+import type { LayoutTokens } from '@fluentui-react-native/tokens';
+import type { TextProps } from '@fluentui-react-native/text';
+
+export const dividerName = 'Divider';
+
+export const DividerInsetSizes = [0, 16, 56, 68, 72, 108] as const;
+export type DividerInsetSize = (typeof DividerInsetSizes)[number];
+export type DividerAlignment = 'start' | 'center' | 'end';
+export type DividerAppearance = 'default' | 'subtle' | 'brand' | 'strong';
+
+export interface DividerProps {
+  /**
+   * If a text or icon is passed, this dictates where content appears in the divider: at the start, centered, or towards the end.
+   * @default 'center'
+   */
+  alignContent?: DividerAlignment;
+  /**
+   * If no color is passed, the divider and its content are colored using different theme tokens depending on the value of this prop.
+   * @default 'default'
+   */
+  appearance?: DividerAppearance;
+  /**
+   * Colors both the divider lines and the content itself, overriding the colors provided by the appearance prop.
+   */
+  color?: ColorValue;
```

Comment:
Curious why this is a prop instead of a token, do we expect that people will want an easier way to change the color of the divider? The web version doesn't have color as a prop.

Reply:
This is leftover from the current Separator, which takes in a color as a prop, but this can also easily be a token to align with the web.

Reply:
I would align to web API. We can add the color prop back if we get feedback that it's a significant problem.

Reply:
We should remove the color prop for now